src/components/shared/StackSettings.css

```diff
@@ -8,6 +8,8 @@
   flex-flow: row nowrap;
   align-items: stretch;
   padding: 0;
+  overflow-x: auto;
```

Comment:
Same comment here, I was mistaken and this should be changed to `overflow: auto hidden;`. Thanks!

Reply:
I'll go ahead and get that reverted. No problem!